lib/pengine/variant.h

```diff
@@ -85,7 +85,18 @@ typedef struct
 
 } container_port_t;
 
+enum container_type {
+        CONTAINER_TYPE_UNKNOWN,
+        CONTAINER_TYPE_DOCKER,
+        CONTAINER_TYPE_RKT
+};
+
+#define CONTAINER_TYPE_UNKNOWN_S "unknown"
+#define CONTAINER_TYPE_DOCKER_S  "docker"
+#define CONTAINER_TYPE_RKT_S     "rkt"
```

Comment:
It's a good idea to prefix new library symbols with PE_ to help avoid naming conflicts